content/getting-started/sections/installing-atom.md

```diff
@@ -86,25 +105,55 @@ $ sudo dpkg -i atom-amd64.deb
 $ sudo apt-get -f install
 ```
 
-##### RedHat and CentOS (yum)
+##### RedHat and CentOS (yum) or Fedora (dnf)
```

Comment:
Is it OK to merge these sections since the instructions are very similar?

Reply:
Yes, it appears to be clear enough.